HW04/aleksei_zhuchkov/spec/homework_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require_relative '../lib/mentor'
+require_relative '../lib/notification'
+require_relative '../lib/student'
+require_relative '../lib/homework'
+
+RSpec.describe Homework do
+  let(:student) { Student.new(name: 'student_name', surname: 'student_surname') }
+  let(:mentor) { Mentor.new(name: 'mentor_name', surname: 'mentor_surname') }
+  let(:test_homework) { described_class.new('hw_title', 'hw_description', student, mentor) }
+
+  describe 'check information about student and mentor' do
+    it 'check right student name' do
+      expect(test_homework.student.name).to eq('student_name')
+    end
+
+    it 'check right student surname' do
+      expect(test_homework.student.surname).to eq('student_surname')
+    end
+
+    it 'check right mentor name' do
+      expect(test_homework.mentor.name).to eq('mentor_name')
+    end
+
+    it 'check right mentor surname' do
+      expect(test_homework.mentor.surname).to eq('mentor_surname')
+    end
+  end
+
+  describe 'checked method print' do
```

Comment:
```suggestion
  describe '#print_homework' do
```

Reply:
I have done it
https://github.com/labs-ruby/homeworks-2022/pull/122/commits/3483c4bde37292ce6fb967100b433f4ba9e659f6